### PR TITLE
Add example for browser upload

### PR DESCRIPTION
**Pull request by shomix**
Readme: https://github.com/spheronFdn/sdk/tree/SPH-1550-example/examples/browser-upload

### CODE REVIEW

**Review by rekpero — CHANGES_REQUESTED** [review]
everything lgtm, just update a small thing in the readme

File: packages/browser-upload/README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ const uploadResult = await upload(<FILES_YOU_WANT_TO_UPLOAD>, token);
 ```
 
 Using this flow, you can control who can use you API token and upload data from your web app.
+Checkout the [LINK](https://github.com/spheronFdn/sdk/tree/SPH-1550-example/examples/browser-upload) for a working example.
```
Comment (rekpero) [inline packages/browser-upload/README.md:85]:
```suggestion
> Checkout the [LINK](https://github.com/spheronFdn/sdk/tree/SPH-1550-example/examples/browser-upload) for a working example.
```

**Review by rekpero — CHANGES_REQUESTED** [review]
everything lgtm, just update a small thing in the readme

File: examples/browser-upload/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+This directory has a simple example on how to use the [@spheron/browser-upload](https://www.npmjs.com/package/@spheron/browser-upload) package with [@spheron/storage](https://www.npmjs.com/package/@spheron/storage).
```
Comment (rekpero) [inline examples/browser-upload/README.md:1]: Add this title before this line

`# Browser Upload Example`

File: examples/browser-upload/client/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## **client**
```
Comment (rekpero) [inline examples/browser-upload/client/README.md:1]:
```suggestion
# Browser Upload Client Example
```

File: examples/browser-upload/server/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+- A express server that has one endpoint `GET: /initiate-upload`, that is used by the **client** project to get the `uploadToken` and to use it to upload data.
```
Comment (rekpero) [inline examples/browser-upload/server/README.md:1]: Add this before this line 
`# Browser Upload Server Example`

**Review by rekpero — CHANGES_REQUESTED** [review]
update these as well

File: packages/browser-upload/README.md (hunk 1 above)
Comment (shomix) [reply packages/browser-upload/README.md]: Updated

File: examples/browser-upload/README.md (hunk 2 above)
Comment (shomix) [reply examples/browser-upload/README.md:3]: Done

File: examples/browser-upload/client/README.md (hunk 3 above)
Comment (shomix) [reply examples/browser-upload/client/README.md]: Done

File: examples/browser-upload/server/README.md (hunk 4 above)
Comment (shomix) [reply examples/browser-upload/server/README.md:3]: Done

**Pull request merged**